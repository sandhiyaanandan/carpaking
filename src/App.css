body
{
  min-width: 320px;
}
.parkingArea
{
  background-color:black;
  width:40%;
  height: 100%;
  margin-left:30%;
  padding-left:2%;

}
.loginBox
{
  border: 10px solid red;
  margin-left: 35%;
  margin-top: 0px;
  width: 10rem;
  padding: 5%;
  background-color: yellow;
}
.submitButton
{
  margin-left: 30%;
  font-size: 80% !important;
  font-weight: bold;
}
.pageHeader
{
  margin-top:0%;
  text-align:center;
  margin-left:5%;
}
.header
{
  margin-top: 2px;
  border: 1px solid blue;
  background-color: blue;
  color: white;
  width: 100%;
  margin-bottom:0%;
}

.content
{
  background-color: red;
}

.box
{
  width:15%;
  height:50px;
  border: 2% solid white;
  background-color: white;
  margin: 2%;

}

ul{
  list-style:none; 
  padding-bottom: 2%;
  margin-left:0px;
}
