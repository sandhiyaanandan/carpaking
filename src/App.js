import React from 'react';
import Registeration from './registeration';
import Parking from './parking';
import Login from './login';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import './App.css';

function App() {
  return (
    <div data-test='Parking-Header'>
    	<h1 className="header">Car Parking</h1>
      <Router>
       <div>	
        	<Switch>
              <Route exact path="/" component={Login}/>
          		<Route path='/Login' component={Login}></Route>
          		<Route path='/Registeration' component={Registeration}></Route>
          		<Route path='/Parking' component={Parking}></Route>
        	</Switch>
        </div>
      </Router>
    </div>
  );
}

export default App;