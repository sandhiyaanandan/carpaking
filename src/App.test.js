import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import {findByTestAttr} from './../jestutils';
import toJson from 'enzyme-to-json';

const setUp =()=>{
	const component = shallow(<App />);
	return component;
};

describe('App', ()=>{
	let wrapper;
	beforeEach(() =>{
		wrapper = setUp();
	});
	
	it('should render the App Component correctly', ()=>{   
    	expect(toJson(wrapper)).toMatchSnapshot();
  	});

	it('should render App component correctly', ()=>{
		let comp = findByTestAttr(wrapper, 'Parking-Header');
		expect(comp.length).toBe(1);
	});
});




