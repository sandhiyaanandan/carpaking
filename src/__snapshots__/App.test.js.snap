// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App Component correctly 1`] = `
<div
  data-test="Parking-Header"
>
  <h1
    className="header"
  >
    Car Parking
  </h1>
  <BrowserRouter>
    <div>
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          path="/Login"
        />
        <Route
          component={[Function]}
          path="/Registeration"
        />
        <Route
          component={[Function]}
          path="/Parking"
        />
      </Switch>
    </div>
  </BrowserRouter>
</div>
`;
